lib/edoc/src/edoc.erl

```diff
@@ -719,20 +748,19 @@ check_forms(Fs, Name) ->
     lists:foreach(Fun, Fs).
 
 
-%% @spec get_doc(File::filename()) -> {ModuleName, edoc_module()}
+%% The EDoc documentation data for a module,
```

Comment:
This is the description of the type <code>edoc_module()</code> - it seems a bit lost here. It should be in the same place as this type is defined.